src/__tests__/createActions-test.js

```diff
@@ -0,0 +1,29 @@
+import { createActions } from '../';
+
+describe('createActions()', () => {
+  describe('resulting action creator map', () => {
+    const types = {
+      ACTION: 'ACTION',
+      MY_LONG_ACTION: 'MY_LONG_ACTION'
+    };
+
+    it(`apply a default string transformation function to action types
+        when mapping to action creators`, () => {
+      const actionCreators = createActions(types);
+
+      expect(actionCreators).to.have.property('action');
+      expect(actionCreators).to.have.property('myLongAction');
+    });
```

Comment:
>Is this the intended behavior?

What 's confusing you here?

The reason I'm asking is that your camelize function behaves unexpectedly and surprisingly. I'd think this from the point of view of a library developer as well as a library consumer.

It looks like if I defined an action "myActionType", it would be translated to "myactiontype". This is strange, given the behavior for uppercase strings, and given that your function is called "camelize" (it should do nothing to an already camelized string). I shouldn't have to pass in my own "transformation" function, it should just work.

This seems to be one of several limitations with this pull request, IMO. Anything else I can help clarify for you?